<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>HNS_Page_Settings_Page</class>
 <widget class="QWidget" name="HNS_Page_Settings_Page">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1140</width>
    <height>800</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="PB_PageSet_Back">
   <property name="geometry">
    <rect>
     <x>820</x>
     <y>640</y>
     <width>163</width>
     <height>74</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Lato</family>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
	background-color:#FFFFFF;
	color:#414141;
	border-style:outset;
	border-width:3px;
	border-color:#1A3760;
	border-radius:12px;
}</string>
   </property>
   <property name="text">
    <string>BACK</string>
   </property>
  </widget>
  <widget class="QToolButton" name="PB_PageSet_BottomJustification">
   <property name="geometry">
    <rect>
     <x>750</x>
     <y>320</y>
     <width>110</width>
     <height>110</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Lato</family>
    </font>
   </property>
   <property name="text">
    <string>...</string>
   </property>
   <property name="icon">
    <iconset resource="../HNS_Connect_UI/resources.qrc">
     <normaloff>:/icons/Bottom Page Justify.png</normaloff>:/icons/Bottom Page Justify.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>60</width>
     <height>60</height>
    </size>
   </property>
   <property name="checkable">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="PB_PageSet_Save">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>640</y>
     <width>163</width>
     <height>74</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Lato</family>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
	background-color:#1A3760;
	color:#FFFFFF;
	border-radius:12px;
}</string>
   </property>
   <property name="text">
    <string>SAVE</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_29">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>100</y>
     <width>181</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Lato</family>
     <pointsize>19</pointsize>
    </font>
   </property>
   <property name="text">
    <string>PAGE OFF TIME</string>
   </property>
  </widget>
  <widget class="QToolButton" name="PB_PageSet_CenterLineJustification">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>320</y>
     <width>110</width>
     <height>110</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Lato</family>
    </font>
   </property>
   <property name="text">
    <string>...</string>
   </property>
   <property name="icon">
    <iconset resource="../HNS_Connect_UI/resources.qrc">
     <normaloff>:/icons/Center Line Justify.png</normaloff>:/icons/Center Line Justify.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>60</width>
     <height>60</height>
    </size>
   </property>
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_32">
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>280</y>
     <width>241</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Lato</family>
     <pointsize>19</pointsize>
    </font>
   </property>
   <property name="text">
    <string>PAGE JUSTIFICATION</string>
   </property>
  </widget>
  <widget class="QToolButton" name="PB_PageSet_RightJustification">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>320</y>
     <width>110</width>
     <height>110</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Lato</family>
    </font>
   </property>
   <property name="text">
    <string>...</string>
   </property>
   <property name="icon">
    <iconset resource="../HNS_Connect_UI/resources.qrc">
     <normaloff>:/icons/Right Line Justify.png</normaloff>:/icons/Right Line Justify.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>60</width>
     <height>60</height>
    </size>
   </property>
   <property name="checkable">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="PB_PageSet_Cancel">
   <property name="geometry">
    <rect>
     <x>650</x>
     <y>640</y>
     <width>163</width>
     <height>74</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Lato</family>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
	background-color:#FFFFFF;
	color:#414141;
	border-style:outset;
	border-width:3px;
	border-color:#1A3760;
	border-radius:12px;
}</string>
   </property>
   <property name="text">
    <string>CANCEL</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_31">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>280</y>
     <width>241</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Lato</family>
     <pointsize>19</pointsize>
    </font>
   </property>
   <property name="text">
    <string>LINE JUSTIFICATION</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_30">
   <property name="geometry">
    <rect>
     <x>580</x>
     <y>100</y>
     <width>181</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Lato</family>
     <pointsize>19</pointsize>
    </font>
   </property>
   <property name="text">
    <string>PAGE ON TIME</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="LE_PageSet_PageOn">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>150</y>
     <width>320</width>
     <height>110</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Lato</family>
     <pointsize>45</pointsize>
    </font>
   </property>
   <property name="text">
    <string>2.0 Sec</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="LE_PageSet_PageOff">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>150</y>
     <width>320</width>
     <height>110</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Lato</family>
     <pointsize>45</pointsize>
    </font>
   </property>
   <property name="text">
    <string>0.0 Sec</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QToolButton" name="PB_PageSet_LeftJustification">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>320</y>
     <width>110</width>
     <height>110</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Lato</family>
    </font>
   </property>
   <property name="text">
    <string>...</string>
   </property>
   <property name="icon">
    <iconset resource="../HNS_Connect_UI/resources.qrc">
     <normaloff>:/icons/Left Line Justify.png</normaloff>:/icons/Left Line Justify.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>60</width>
     <height>60</height>
    </size>
   </property>
   <property name="checkable">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_28">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>20</y>
     <width>611</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Lato</family>
     <pointsize>33</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color:#1A3760</string>
   </property>
   <property name="text">
    <string>PAGE DEFAULTS SETTINGS</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QToolButton" name="PB_PageSet_CenterPageJustification">
   <property name="geometry">
    <rect>
     <x>640</x>
     <y>320</y>
     <width>110</width>
     <height>110</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Lato</family>
    </font>
   </property>
   <property name="text">
    <string>...</string>
   </property>
   <property name="icon">
    <iconset resource="../HNS_Connect_UI/resources.qrc">
     <normaloff>:/icons/Center Page Justify.png</normaloff>:/icons/Center Page Justify.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>60</width>
     <height>60</height>
    </size>
   </property>
   <property name="checkable">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="PB_PageSet_PickFont">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>460</y>
     <width>163</width>
     <height>107</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Lato</family>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
	background-color:#FFFFFF;
	color:#414141;
	border-style:outset;
	border-width:3px;
	border-color:#1A3760;
	border-radius:12px;
}</string>
   </property>
   <property name="text">
    <string>DEFAULT
FONT</string>
   </property>
  </widget>
  <widget class="QToolButton" name="PB_PageSet_TopJustification">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>320</y>
     <width>110</width>
     <height>110</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Lato</family>
    </font>
   </property>
   <property name="text">
    <string>...</string>
   </property>
   <property name="icon">
    <iconset resource="../HNS_Connect_UI/resources.qrc">
     <normaloff>:/icons/Top Page Justify.png</normaloff>:/icons/Top Page Justify.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>60</width>
     <height>60</height>
    </size>
   </property>
   <property name="checkable">
    <bool>true</bool>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="../HNS_Connect_UI/resources.qrc"/>
 </resources>
 <connections/>
</ui>
